cpp/net/url_fetcher.cc

```diff
@@ -162,7 +177,7 @@ void State::RequestDone(evhttp_request* req) {
     return;
```

Comment:
I don't actually know, would be lovely to be able to remove all that stuff... but that's a story for another PR I think.